notebooks/JWST/NIRCam/PSF_phot_01/GriddedPSFModel_simulation_and_photometry.ipynb

```diff
@@ -0,0 +1,552 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we'll create an Astropy table defining the initial guesses of star positions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "init_tbl = Table()\n",
+    "init_tbl['x_0'] = xx.astype(int)\n",
+    "init_tbl['y_0'] = yy.astype(int)\n",
+    "init_tbl['flux_0'] = zz.astype(int)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then, we define the parameters to initialize a `BasicPSFPhotometry` instance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sigma_psf = 3.\n",
+    "daogroup = DBSCANGroup(2.0 * sigma_psf * gaussian_sigma_to_fwhm)\n",
+    "mmm_bkg = MMMBackground()\n",
+    "fit_shape = (eval_yshape, eval_xshape)\n",
+    "phot = BasicPSFPhotometry(daogroup, mmm_bkg, psfmodel, fit_shape, finder=None, \n",
+    "                          aperture_radius=3.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we call the instance on the data. This step might take a few seconds. See https://github.com/astropy/photutils/issues/631 for more information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tbl = phot(data, init_guesses=init_tbl)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result is an Astropy table containing the initial and fitted values of the star positions and fluxes. We can format the displayed values by setting `table[column_name].format`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tbl['x_fit'].format = '%.1f'\n",
+    "tbl['y_fit'].format = '%.1f'\n",
+    "tbl['flux_fit'].format = '%.4e'\n",
+    "tbl['flux_unc'].format = '%.4e'\n",
+    "tbl['x_0_unc'].format = '%.4e'\n",
+    "tbl['y_0_unc'].format = '%.4e'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tbl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also view the residual image of the best-fit model PSF image subtracted from the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "diff = phot.get_residual_image()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(9, 9))\n",
+    "plt.imshow(diff, norm=norm, origin='lower')\n",
```

Comment:
shouldn't this use its own `norm` object, since the data are different from the previous one?